.github/workflows/pr.yaml

```diff
@@ -46,3 +48,20 @@ jobs:
     uses: rapidsai/shared-action-workflows/.github/workflows/conda-python-tests.yaml@main
     with:
       build_type: pull-request
+  wheel-build:
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux-build.yml@upkeep/manylinux-improvements
+    with:
+      build_type: pull-request
+      package-dir: python
+      package-name: rmm
+      skbuild-configure-options: "-DRMM_BUILD_WHEELS=ON"
```

Comment:
It annoys me that we have to specify this in every repo, but I don't see a great WAR at this stage. For now, just making a note that this is one more bit of boilerplate that it would be nice to get rid of somehow.